### ISSUE TITLE
AmbiguousTimeError when filtering DST day

### ISSUE THREAD

**Issue by andreroggeri** [carltongibson/django-filter#633]
I found this issue this week because where I live the DST ends 2017-02-18 23:00.

I think the issue is on the `utils.py`

```
def handle_timezone(value):
    if settings.USE_TZ and timezone.is_naive(value):
        return timezone.make_aware(value, timezone.get_default_timezone())
    elif not settings.USE_TZ and timezone.is_aware(value):
        return timezone.make_naive(value, timezone.UTC())
    return value
```

`make_aware` accepts a parameter named `is_dst` but it isn't used.

I think it's necessary to check if the timezone is on DST and pass the parameter to `make_aware`

**Comment by carltongibson** [comment]
OK. Good. Time zones. 😤 

First thing is a few test cases. We need some example datetimes where DST is active/inactive and we're getting the wrong result here. 

(Ideally those can be embedded in a test case but just a shell transcript is fine to begin.)

Time zones. 😤

**Comment by andreroggeri** [comment]
I'll see what I can do. :)

**Comment by andreroggeri** [comment]
OK, I made a very simple testcase that triggers the error(It doesn't have any assert yet, I need to figure out what is the expected result). I'll try to dig more tomorrow to see if I can make a fix.

Maybe [this answer on SO](http://stackoverflow.com/questions/19774709/use-python-to-find-out-if-a-timezone-currently-in-daylight-savings-time) can help.

**Comment by carltongibson** [comment]
Closed by #634
